srcopsmetrics/entities/release.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2020 Dominik Tuchyna
+#
+# This file is part of thoth-station/mi - Meta-information Indicators.
+#
+# thoth-station/mi - Meta-information Indicators is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# thoth-station/mi - Meta-information Indicators is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with thoth-station/mi - Meta-information Indicators.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Release entity class."""
+
+import logging
+from typing import Any, List, Union
+
+from github.GitRelease import GitRelease
+from github.GitTag import GitTag
+from semver import VersionInfo
+from voluptuous.schema_builder import Schema
+
+from srcopsmetrics.entities import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Release(Entity):
```

Comment:
This class need to be added to EntityTypeEnum to be used with `-e` in CLI

Reply:
no need for that, with the new dynamic import the class can be used with -e option
(see this here https://github.com/thoth-station/mi/pull/186#discussion_r501320326 )